test/js/keyed-functions.js

```diff
@@ -38,6 +38,10 @@ function getEnv() {
   return process.env
 }
 
+function logsSomething() {
+  console.log("Something")
```

Comment:
We should update this to return a value so that we can ensure that it works as intended. Which is a function calling `console.log` a few times, but we are still able to get the return value.

Reply:
Good idea, I'll change the test

Reply:
The test now returns (and asserts) `42` and the js now also prints to `process.stdout` to make sure that it doesn't interfere either